docs/Containerfile

```diff
@@ -0,0 +1,14 @@
+# The resulting image can be used to
+# generate podman-remote documentation.
+FROM golang

```

Comment:
please use the full name, we should really avoid short names in general as best practise

Reply:
Done